ChopChop/ViewModels/OnlineRecipes/OnlineRecipeViewModel.swift

```diff
@@ -4,17 +4,24 @@ import Combine
 class OnlineRecipeViewModel: ObservableObject {
     private(set) var recipe: OnlineRecipe
 
+    private var creatorCancellable: AnyCancellable?
     private var recipeCancellable: AnyCancellable?
     private var followeesCancellable: AnyCancellable?
     private var firstRaterCancellable: AnyCancellable?
     private var imageCancellable: AnyCancellable?
     let storageManager = StorageManager()
 
+    @Published var creatorName = "No name" {
+        willSet { self.objectWillChange.send() }
+    }
```

Comment:
Why is there a need to manually send a change notification?

Reply:
This was added in a previous commit by another person, and its removal doesn't seem to break anything, so I don't know the purpose of this. @shadowezz @chrystalquek any inputs? If not I will remove it

Reply:
I think need to ask chrystal on this